modules/core/src/viewports/globe-viewport.ts

```diff
@@ -59,37 +62,39 @@ export type GlobeViewportOptions = {
 };
 
 export default class GlobeViewport extends Viewport {
-  // @ts-ignore
-  longitude: number;
-  // @ts-ignore
-  latitude: number;
-  resolution: number;
+  longitude!: number;
+  latitude!: number;
+  resolution!: number;
 
   constructor(opts: GlobeViewportOptions = {}) {
     const {
       latitude = 0,
       longitude = 0,
       zoom = 0,
-      nearZMultiplier = 0.1,
-      farZMultiplier = 2,
+      nearZMultiplier = 0.5,
+      farZMultiplier = 1,
       resolution = 10
     } = opts;
 
-    let {height, altitude = 1.5} = opts;
+    let {height, altitude = 1.5, fovy} = opts;
 
     height = height || 1;
-    altitude = Math.max(0.75, altitude);
+    if (fovy) {
```

Comment:
This aligns us more with other views, right?
We always support fovy and altitude is a backup for geospatial views?

Reply:
That is correct.
`altitude` is a parameter we inherited from the mapbox/maplibre. The name is a bit misleading. It refers to the camera's distance to the sea plane.